scripts/lib/build-pgsearch

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+tmpdir="$(mktemp -d)"
+trap 'rm -r "$tmpdir"' EXIT
+cd "$tmpdir"
+
+curl -fLO --retry 3 "https://github.com/paradedb/paradedb/releases/download/v0.10.0/postgresql-14-pg-search_0.10.0-1PARADEDB-jammy_amd64.deb"
+
+dpkg -i "postgresql-14-pg-search_0.10.0-1PARADEDB-jammy_amd64.deb"
```

Comment:
You should upgrade to 0.10.3. There's some really serious performance improvements (up to 100x faster queries). We're releasing `v0.11.0` on Tuesday October 15, and it'll have even more improvements